### PR TITLE
Rework the traefik example

### PR DESCRIPTION
**Pull request by hunhejj**
Make the Traefik example more complete.

### CODE REVIEW

File: reverse-proxy.md [hunk 1]
```diff
@@ -354,11 +354,34 @@ Of course you need to modify `<your-nc-domain>` to the domain on which you want
 
 <summary>click here to expand</summary>
 
-**Disclaimer:** It might be possible that the config below is not working 100% correctly, yet. Improvements to it are very welcome!
```
Comment (szaimen) [inline reverse-proxy.md:357]: Can we please keep this?

**Review by szaimen — COMMENTED** [review]
Looks already pretty good to me, thank you! :) Only one small remark.

**Comment by hunhejj** [comment]
Sorry, I didn't see your earlier message. I will update the MR in the evening!

**Comment by szaimen** [comment]
> Sorry, I didn't see your earlier message. I will update the MR in the evening!

Thank you! :)

**Comment by hunhejj** [comment]
Adjusted it @szaimen.

**Review by szaimen — APPROVED** [review]
LGTM, thanks! :)

**Pull request merged**